src/js/workshop-comments.js

```diff
@@ -0,0 +1,144 @@
+const commentForm = document.getElementById("comment-form");
+
+const nameDOMElement = document.getElementById("name");
+const commentDOMElement = document.getElementById("comment");
+
+commentForm.addEventListener("submit", (event) => {
+	event.preventDefault();
+
+	const nameDOMElement = document.getElementById("name");
```

Comment:
Sorry that in our pair programming it was me who started to use the all-capitalized "DOM" as a part of the variable name. After thinking more about it, I felt it might be more conventional to have the variable named "nameDomElement". Could you please rename:

1. nameDOMElement -> nameDomElement
2. commentDOMElement -> commentDomElement

Thanks.